benchmark-framework/pom.xml

```diff
@@ -145,6 +140,12 @@
             <artifactId>HdrHistogram</artifactId>
             <version>2.1.12</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>driver-nats-streaming</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Was this change intended?

Reply:
Hi @merlimat, yes this change is intended. I tried to explain the justification in the PR description:
`Set scope provided to driver-nats-streaming. The reason for doing that is that driver-nats-streaming is transitively importing a very old version of Google Protobuf (3.9.1 - Aug 06, 2019). Pravega also uses Google Protobuf. When upgrading Pravega, we found a NoSuchMethodError related to Google Protobuf and the reason was that the newer Pravega version are compiled with a more recent Google Protobuf version with incompatible changes (since version 3.17.2). Having both Google Protobuf versions in the classpath leads to runtime errors, so I decided to prevent the oldest Google Protobuf version to be transitively imported by driver-nats-streaming. Given that NATS Streaming has been deprecated, I feel that it is a safe action to do (which can precede the removal of NATS Streaming until NATS JetStream is available).`